src/maintainer/maintainer_faq.rst

```diff
@@ -110,4 +110,9 @@ This message can be safely ignored as conda-forge doesn't use conda-verify.
 
     The exit code 139 itself actually is the general exit code for a segmentation fault. This could also mean that you have run into a different issue but the above issue is the most likely one with our CentOS 6-based images.
 
+.. _mfaq_package_submit:
+
+:ref:`(Q) <mfaq_package_submit>` **Is it necessary for me to be a maintainer of the package I submit to Conda-forge?**
```

Comment:
```suggestion
:ref:`(Q) <mfaq_package_submit>` **Is it necessary for me to be an upstream maintainer of the package I submit to Conda-forge?**
```
Is there a standard nomenclature here? Upstream seems common but original is also sometimes used. @viniciusdc

Reply:
afaik no, but I think that the upstream addition took care of the problem already :smile: